script.js

```diff
@@ -97,20 +85,16 @@ function renderRecipe() {
     recipeInnerModal.appendChild(link)
 
     recipeModal.style.display = 'flex'
-
 }
 
-
-
-
+getRecipeBtn.addEventListener("click", renderRecipe)
```

Comment:
I moved this down here from the top of the file so that it's beside the `renderRecipe` function as they're closely related to each other.